tests/Meilisearch.Tests/SettingsTests.cs

```diff
@@ -468,9 +468,9 @@ private async Task AssertGetInequality<TValue>(IndexGetMethod<TValue> getMethod,
 
         private async Task AssertTaskInfoSucceeded(TaskInfo task)
         {
-            task.Uid.Should().BeGreaterThan(0);
-            task = await _index.WaitForTaskAsync(task.Uid);
-            task.Status.Should().Be(TaskInfoStatus.Succeeded);
+            task.TaskUid.Should().BeGreaterThan(0);
+            var taskRessource = await _index.WaitForTaskAsync(task.TaskUid);
```

Comment:
Why not just `task`? I believe the only protection is when we call `Task` with a capital letter.
```suggestion
            var task = await _index.WaitForTaskAsync(task.TaskUid);
```

Reply:
Because the function parameter is already called task. This is not a problem?

Reply:
yep, that's a big problem hahaha 😬